packages/uploadthing/src/client.ts

```diff
@@ -102,14 +104,14 @@ export type UploadFileResponse = {
 
 export const DANGEROUS__uploadFiles = async <TRouter extends FileRouter>(
   opts: UploadFilesOptions<TRouter>,
-  config?: {
-    url?: string;
+  config: {
+    url: string;
   },
```

Comment:
would it make sense to collapse this into a single parameter and put the url in the opts object?

Reply:
I think we had it as a config object for future expansion reasons? But you're right it does seem kind of silly